ci/jenkins/test-mc.sh

```diff
@@ -247,19 +265,24 @@ function deliver_antrea_multicluster {
 
     DOCKER_REGISTRY="${DOCKER_REGISTRY}" ./hack/build-antrea-linux-all.sh --pull
     echo "====== Delivering Antrea to all the Nodes ======"
-    docker save -o ${WORKDIR}/antrea-ubuntu.tar $DOCKER_REGISTRY/antrea/antrea-ubuntu:latest
-
+    docker save -o ${WORKDIR}/antrea-ubuntu.tar ${DOCKER_REGISTRY}/antrea/antrea-ubuntu:latest
 
     for kubeconfig in "${multicluster_kubeconfigs[@]}"
     do
-       kubectl get nodes -o wide --no-headers=true ${kubeconfig}| awk '{print $6}' | while read IP; do
-            rsync -avr --progress --inplace -e "ssh -o StrictHostKeyChecking=no" "${WORKDIR}"/antrea-ubuntu.tar jenkins@[${IP}]:${WORKDIR}/antrea-ubuntu.tar
-            if ${IS_CONTAINERD};then
-              ssh -o StrictHostKeyChecking=no -n jenkins@${IP} "${CLEAN_STALE_IMAGES}; sudo ctr -n=k8s.io images import ${WORKDIR}/antrea-ubuntu.tar" || true
-            else
-              ssh -o StrictHostKeyChecking=no -n jenkins@${IP} "${CLEAN_STALE_IMAGES}; docker load -i ${WORKDIR}/antrea-ubuntu.tar" || true
-            fi
-       done
+        if [[ ${KIND} == "true" ]]; then
+            for name in ${CLUSTER_NAMES[*]}; do
+                kind load docker-image  ${DOCKER_REGISTRY}/antrea/antrea-ubuntu:latest --name ${name}
```

Comment:
It seems the command will be execute repeatedly, N*N. should the loop be out of the kubeconfig loop?

Reply:
Fixed.